src/main/java/uk/gov/justice/probation/courtcasematcher/messaging/model/commonplatform/CPHearingType.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.probation.courtcasematcher.messaging.model.commonplatform;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PRIVATE, force = true)
+public class CPHearingType {
+  private final String description;
```

Comment:
You could move this string type to CPHearing.java itself, no?

Reply:
It is the based on the source JSON model